packages/react-admin-rte/src/core/Controls/ControlButton.tsx

```diff
@@ -0,0 +1,19 @@
+import { SvgIconProps } from "@material-ui/core/SvgIcon";
+import * as React from "react";
+import * as sc from "./ControlButton.sc";
+
+interface IProps {
+    disabled?: boolean;
+    selected?: boolean;
+    onButtonClick?: (e: React.MouseEvent) => void;
+    label?: string;
+    Icon?: (props: SvgIconProps) => JSX.Element;
+}
+
+export default function ControlButton({ disabled = false, selected = false, onButtonClick, label = "", Icon }: IProps) {
```

Comment:
die componente bitte in src/index.ts exportieren, die braucht man auch für RTE-Erweiterungen ausserhalb der Library:

```
export { default as ControlButton, IProps as IControlButtonProps } from "./core/Controls/ControlButton";

```